app/services/transfer_assignments_service.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class TransferAssignmentsService
+  attr_accessor :organization, :old_user, :new_user
```

Comment:
Can we add a newline after this, to be more consistent with the app?

Reply:
done 👍🏼